charts/tigera-operator/values.yaml

```diff
@@ -38,6 +38,10 @@ certs:
     commonName:
     caBundle:
 
+# Whether or not the tigera/operator should manange CustomResourceDefinitions
+# needed to run itself and Calico. If disabled, you must manage these resources out-of-band.
+manageCRDs: true
+
```

Comment:
is there any special consideration needed for OCP (which as far as I can tell, should always have `-manage-crds=true`)?

Reply:
Don't think so!